### PR TITLE
Updated README.

### PR DESCRIPTION
**Pull request by jebrodsky**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,132 @@
...
+following:
+
+-   Alignment of the standard’s principles with academic content
+
+-   Flexibility and potential to be customized for different instructional settings
+
+-   Articulation of competencies that, when paired with learning activities, can be measured for mastery
+
+As part of this release of the DLCC, we have prototyped a representation of the [**ACRL’s 2016 Information Literacy Standard**](http://www.ala.org/acrl/standards/ilframework). As the broadest and most conceptual of the frameworks our faculty have worked with, we have found the ACRL Standard most helpful for faculty who are able to integrate the information literacy thresholds and their digital skill components into their instruction of disciplinary conventions. We will be excited to see how others break these thresholds down along the Bloom’s scale and pair them with activities to create discrete competencies.
+
```
Comment (nbuonin) [inline README.md]: I'm not sure you need 'the' here: 

> We will be excited to see how others break these thresholds down along the Bloom’s scale and pair them with activities to create discrete competencies.

File: README.md [hunk 2]
```diff
@@ -1 +1,132 @@
...
+6.  Go to the “csv” folder inside the “images” folder in your repository. Drag the file “standards.csv” into the “csv” folder. It should replace the existing “standards.csv” file.
+
+7.  Describe your change and create a new branch for this commit and start a pull request. ([**What is a pull request?**](https://help.github.com/articles/about-pull-requests/))
+
+![Screenshot of proposing a change in GitHub](media/images/Propose_Changes_GitHub.png)
+
+8.  Go to the “index.html” file in your repository and edit the Title (line 203) and descriptions (line 206) to suit your context. Describe your change and create a new branch for this commit and start a pull request.
+
+    a.  Note: All other descriptions (for lenses, pop up box copy, etc.) are determined by your “standards.csv” file. To make further changes to description, you must make changes in your Google Sheet and upload a new “standards.csv” file with your changes. We discourage editing in the standards.csv file directly.
+
```
Comment (nbuonin) [inline README.md]: I think this could read: "To make further changes to **the** description,..."

Comment (jebrodsky) [reply README.md]: Sounds good.

File: README.md (hunk 1 above)
Comment (jebrodsky) [reply README.md]: Sounds good.

**Pull request merged**